GDJS/Runtime/InAppTutorialMessage.tsx

```diff
@@ -0,0 +1,185 @@
+const redHeroImage =
```

Comment:
Put this into the namespace, otherwise the esbuild transpilation will create this as a global variable.

(we're sadly not using modules)